ffd_info_exchange/fafsa/templates/fafsa_form.html

```diff
@@ -3,27 +3,6 @@
 
 {% block content %}
   {% load floppyforms %}
-    <header class="usa-site-header disclaimer" role="banner">
-
-      <div class="usa-disclaimer">
-        <div class="usa-grid-no-padding">
-          <div class="usa-width-one-whole">
-            <div class="disclaimer-left">
-                <span class="usa-disclaimer-official">
-                  An official website of the United States Government
-                  <img class="usa-flag_icon" alt="U.S. flag signifying that this is a United States Federal Government website" src="{% static 'assets/img/usa-flag.png' %}">
-                </span>
-              </div>
-              <div class="disclaimer-right">
```

Comment:
Thanks for adding the "here's how you know" section below! I'd like to keep this "This site is a discovery prototype" section in there too, for the sake of context — would you mind adding it back in?